app/views/dashboard/ng-partials/_settings.html.haml

```diff
@@ -20,22 +18,22 @@
             %input{placeholder: "4bbd2jm242dl5wmimbwz4rvlu77m0a4i", type: "text", ng_model: 'form.token', required: true}/
         .row
           .large-12.columns
-            %button.button
-              %i.fi-check{style: 'font-size: 18px;'}
-                Store Settings
-  %table
-    %thead
-      %tr
-        %th{width: '90%'} Friendly Name
-        %th{width: '5%'} Delete
-        %th{width: '5%'} Edit
-    %tbody
-      %tr{'ng-repeat' => 'device in devices.list'}
-        %td
-          {{ device.name }}
-        %td
-          %button.tiny.alert.button{'ng-click' => 'removeDevice(device)'}
-            %i.fi-x{style: 'font-size: 18px;'}
-        %td
-          %button.tiny.button{ng_click: 'selectDevice(device)'}
-            %i.fi-wrench{style: 'font-size: 18px;'}
+            %button.button{class: 'green-button'}
```

Comment:
With haml, you can just write `%button.button.green-button`. The dots signify a class name and can be chained.